src/sender.js

```diff
@@ -142,13 +150,18 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue, channel }) {
     logger.info(`Finished processing msg`)
 
     if (insufficientFunds) {
+      logger.warn(
+        'Insufficient funds. Stop sendint transactions until the account has the minimum balance'
+      )
       channel.close()
       waitForFunds(web3Instance, VALIDATOR_ADDRESS, minimumBalance, resume)
```

Comment:
since this method requests data from the chain it make sense to add logging there in order to see if it stuck or not.

Reply:
Fixed in 453cc4bd1f01c9f2f7eccd2bd7bb84963de03444.